search/src/test/scala/weco/api/search/services/WorksServiceTest.scala

```diff
@@ -149,242 +102,99 @@ class WorksServiceTest
     }
 
     it("only finds results that match a query if doing a full-text search") {
-      val workDodo = indexedWork().title("A drawing of a dodo")
-      val workMouse = indexedWork().title("A mezzotint of a mouse")
-
       assertListOrSearchResultIsCorrect(
-        allWorks = List(workDodo, workMouse),
+        allWorks = List("work-title-dodo", "work-title-mouse"),
         expectedWorks = List(),
         expectedTotalResults = 0,
         worksSearchOptions =
           createWorksSearchOptionsWith(searchQuery = Some(SearchQuery("cat")))
       )
 
       assertListOrSearchResultIsCorrect(
-        allWorks = List(workDodo, workMouse),
-        expectedWorks = List(workDodo),
+        allWorks = List("work-title-dodo", "work-title-mouse"),
+        expectedWorks = List("work-title-dodo"),
         expectedTotalResults = 1,
         worksSearchOptions =
           createWorksSearchOptionsWith(searchQuery = Some(SearchQuery("dodo")))
       )
     }
 
     it("doesn't throw an exception if passed an invalid query string") {
-      val workEmu = indexedWork().title("An etching of an emu")
-
       // unmatched quotes are a lexical error in the Elasticsearch parser
       assertListOrSearchResultIsCorrect(
-        allWorks = List(workEmu),
-        expectedWorks = List(workEmu),
+        allWorks = List("work-title-dodo"),
+        expectedWorks = List("work-title-dodo"),
         expectedTotalResults = 1,
         worksSearchOptions = createWorksSearchOptionsWith(
-          searchQuery = Some(SearchQuery("emu \""))
+          searchQuery = Some(SearchQuery("dodo \""))
         )
       )
     }
 
-    it("returns results in consistent sort order") {
```

Comment:
I binned this test because it's a bit of a faff in the new structure, and I don't think we care about it that much. It's also tested elsewhere.